sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/DataSourceReadBenchmark.scala

```diff
@@ -34,10 +34,15 @@ import org.apache.spark.sql.vectorized.ColumnVector
 
 /**
  * Benchmark to measure data source read performance.
- * To run this:
- *  spark-submit --class <this class> <spark sql test jar>
+ * To run this benchmark:
+ * {{{
+ *   1. without sbt: bin/spark-submit --class <this class> <spark sql test jar>
```

Comment:
Hi, @peter-toth . Could you run this command actually?

Reply:
```
bin/spark-submit --class org.apache.spark.sql.execution.benchmark.DataSourceReadBenchmark --jars core/target/spark-core_2.11-3.0.0-SNAPSHOT-tests.jar,sql/catalyst/target/spark-catalyst_2.11-3.0.0-SNAPSHOT-tests.jar sql/core/target/spark-sql_2.11-3.0.0-SNAPSHOT-tests.jar
```
does work for me, but I checked in `FilterPushdownBenchmark` and it seems we don't mention other required jars.
Shall I modify the command?

Reply:
Yes. We noticed that the required jars are introduced during refactoring. So, we start to fix this guide in recent PRs like [this](https://github.com/apache/spark/commit/669ade3a8eed0016b5ece57d776cea0616417088).